src/main/scala/vee/db/Keys.scala

```diff
@@ -65,48 +42,20 @@ object Keys {
-
-  def approvedFeatures: Key[Map[Short, Int]]  = Key("approved-features", Array[Byte](0, 29), readFeatureMap, writeFeatureMap)
-  def activatedFeatures: Key[Map[Short, Int]] = Key("activated-features", Array[Byte](0, 30), readFeatureMap, writeFeatureMap)
-
-  def dataKeyChunkCount(addressId: BigInt): Key[Int] =
-    Key("data-key-chunk-count", addr(31, addressId), Option(_).fold(0)(Ints.fromByteArray), Ints.toByteArray)
-  def dataKeyChunk(addressId: BigInt, chunkNo: Int): Key[Seq[String]] =
-    Key("data-key-chunk", addr(32, addressId) ++ Ints.toByteArray(chunkNo), readStrings, writeStrings)
-
-  def dataHistory(addressId: BigInt, key: String): Key[Seq[Int]] = historyKey("data-history", 33, addressId.toByteArray ++ key.getBytes(UTF_8))
-  def data(addressId: BigInt, key: String)(height: Int): Key[Option[DataEntry[_]]] =
-    Key.opt("data", hBytes(34, height, addressId.toByteArray ++ key.getBytes(UTF_8)), DataEntry.parseValue(key, _, 0)._1, _.valueBytes)
-
-  def sponsorshipHistory(assetId: ByteStr): Key[Seq[Int]] = historyKey("sponsorship-history", 35, assetId.arr)
-  def sponsorship(assetId: ByteStr)(height: Int): Key[SponsorshipValue] =
-    Key("sponsorship", hBytes(36, height, assetId.arr), readSponsorship, writeSponsorship)
-
-  val addressesForWavesSeqNr: Key[Int]                = intKey("addresses-for-waves-seq-nr", 37)
-  def addressesForWaves(seqNr: Int): Key[Seq[BigInt]] = Key("addresses-for-waves", h(38, seqNr), readBigIntSeq, writeBigIntSeq)
-
-  def addressesForAssetSeqNr(assetId: ByteStr): Key[Int] = bytesSeqNr("addresses-for-asset-seq-nr", 39, assetId.arr)
-  def addressesForAsset(assetId: ByteStr, seqNr: Int): Key[Seq[BigInt]] =
-    Key("addresses-for-asset", hBytes(40, seqNr, assetId.arr), readBigIntSeq, writeBigIntSeq)
+  val addressesForVeeSeqNr: Key[Int]                = intKey("addresses-for-vee-seq-nr", 37)
+  def addressesForVee(seqNr: Int): Key[Seq[BigInt]] = Key("addresses-for-vee", h(38, seqNr), readBigIntSeq, writeBigIntSeq)
 
   def addressTransactionSeqNr(addressId: BigInt): Key[Int] = bytesSeqNr("address-transaction-seq-nr", 41, addressId.toByteArray)
-  def addressTransactionIds(addressId: BigInt, seqNr: Int): Key[Seq[(Int, ByteStr)]] =
-    Key("address-transaction-ids", hBytes(42, seqNr, addressId.toByteArray), readTransactionIds, writeTransactionIds)
+  def addressTransactionIds(addressId: BigInt, seqNr: Int): Key[Seq[ByteStr]] =
+    //Key("address-transaction-ids", hBytes(42, seqNr, addressId.toByteArray), readTransactionIds, writeTransactionIds)
+    Key("address-transaction-ids", hBytes(42, seqNr, addressId.toByteArray), readTxIds, writeTxIds)
 
-  val AliasIsDisabledPrefix: Short = 43
-  def aliasIsDisabled(alias: Alias): Key[Boolean] =
-    Key("alias-is-disabled", bytes(AliasIsDisabledPrefix, alias.bytes.arr), Option(_).exists(_(0) == 1), if (_) Array[Byte](1) else Array[Byte](0))
+  def snapshot(addressId: BigInt)(height: Int): Key[Long] =
```

Comment:
this key is not used right?

Reply:
Not used for now. It may use later for "snapshotAtHeight" in LevelDBWriter